node/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "opent2t-lib",
+  "version": "0.1.0",
+  "description": "Open Translators to Things",
+  "main": "build/lib",
+  "typings": "build/lib",
+  "scripts": {
+    "build": "tsc",
+    "lint": "tslint --force lib/**/*.ts",
+    "test": "tsc && ava --verbose"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/opent2t/opent2t.git"
+  },
+  "author": "opent2t@microsoft.com",
```

Comment:
Currently  most of the OpenT2T package.json files have `"author": ""`, though some have "Microsoft Corp." which is invalid since I think it's supposed to be an email address.

I'll make this blank for now and we can decide what to do with all of them later.

---
In reply to: [71898626](https://github.com/opent2t/opent2t/pull/1#discussion_r71898626) [](ancestors = 71898626)